conf/vertebrates/jira_recurrent_tickets.json

```diff
@@ -169,6 +169,18 @@
             "summary": "Run Sauropsids EPO_low",
             "name_on_graph": "EPO-2X:Sauropsids"
          },
+         {
+            "component": "Production tasks",
+            "description": "relco_doc#Releasecoordinationdocumentation-Mainsite",
+            "summary": "Run Pig breeds EPO",
+            "name_on_graph": "EPO:PigBreeds"
```

Comment:
Use an underscore between "Pig" and "Breeds". The JavaSript will automatically replace it with a space on the graph